src/View/Widget/TimezoneWidget.php

```diff
@@ -31,10 +31,10 @@ protected function _renderContent(array $data): array
     /**
      * Converts list of regions to identifiers list.
      *
-     * @param array $options List of regions
+     * @param array|null $options List of regions
      * @return array
      */
-    protected function _identifierList(array $options): array
+    protected function _identifierList(?array $options = null): array
```

Comment:
There's no need to make the argument nullable, can just change to `(array $options = [])` to avoid having to pass and array always.

Reply:
Hi @ADmad,

The error I got was this one:
`Argument 1 passed to ADmad\I18n\View\Widget\TimezoneWidget::_identifierList() must be of the type array, null given`

My FormHelper is friendsofcake/bootstrap-ui, so maybe I have a special interaction here where the "options" key isn't initialized.
I didn't test with the basic FormHelper.

In my case, with your suggestion, I have to pass empty options in my Form->control() to get it to work:
`<?= $this->Form->control('timezone', ['type' => 'timezone', 'options' => []]) ?>`

Do you think it's only related to friendsofcake/bootstrap-ui ?
I try with a fresh CakePHP 4 to test.

Reply:
Tested on a fresh CakePHP 4.1.1 with only cakephp-i18n, basic config, and the timezone widget for FormHelper.
I get the same error out of the box.

What do you advise?
If passing empty options to a form control is recommended, I can add details in the README.
From my personnal point of view, options can be null on a form control, and so "_identifierList()" should allow that type.